### PR TITLE
tde post

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

File: content/blog/2021/2021-04-02-transparent-data-encryption/index.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+---
+
+Oracle&reg; introduced the Transparent Data Encryption (TDE) feature in Oracle 12C for
+security purposes to let users enable tablespace and column-level encryption for sensitive
+data. 
+
+<!--more-->
+
+### introduction
```
Comment (stephamon) [inline content/blog/2021/2021-04-02-transparent-data-encryption/index.md:28]: Capitalize "Introduction"

Comment (catlook) [reply content/blog/2021/2021-04-02-transparent-data-encryption/index.md:28]: doh!!

File: content/blog/2021/2021-04-02-transparent-data-encryption/index.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+which can be an Oracle software or hardware keystore. This TDE master encryption key encrypts
+and decrypts the TDE table key, which in turn encrypts and decrypts data in the table column."
+
+{{<img src="Picture1.png" title="" alt="">}}
+
+*Image source: [https://docs.oracle.com/database/121/ASOAG/introduction-to-transparent-data-encryption.htm#ASOAG10137](https://docs.oracle.com/database/121/ASOAG/introduction-to-transparent-data-encryption.htm#ASOAG10137)*
+
+</br>
+
+##### 2. Transparent Data Encryption tablespace encryption:
```
Comment (stephamon) [inline content/blog/2021/2021-04-02-transparent-data-encryption/index.md:76]: Remove colon at the end of the header

Comment (catlook) [reply content/blog/2021/2021-04-02-transparent-data-encryption/index.md:76]: !

**Review by stephamon — APPROVED** [review]

**Pull request merged**